StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/UpdatePaymentMethodViewController.swift

```diff
@@ -218,6 +218,12 @@ final class UpdatePaymentMethodViewController: UIViewController {
     }
 
     @objc private func removePaymentMethod() {
+        for element in self.paymentMethodForm.getAllUnwrappedSubElements() {
+            if let textElement = element as? TextFieldElement {
+                textElement.endEditing(true, continueToNextField: false)
+            }
+        }
+
```

Comment:
What's the scenario where the keyboard is presented when a payment method is removed? Shouldn't the taps required to remove the payment method dismiss the keyboard?

Reply:
Ah, good question -- a user can tap any text field (which presents the keyboard), then the user decides they no longer want to edit that field and attempts to delete the payment method while the keyboard is still showing.  This will introduce some rendering jank because UIAlertView dismisses keyboard without re-drawing the screen -- so i've remedied this by forcibly dismissing the keyboard before this happens.